src/js/themes/base.d.ts

```diff
@@ -726,7 +726,14 @@ export interface ThemeType {
     round?: RoundType;
   };
   formField?: {
-    border?: BorderType;
+    border?: {
+      color?: ColorType;
+      position?: BorderType;
+      side?: BorderType;
+      error?: {
+        color?: ColorType;
+      };
+    };
```

Comment:
Wouldn't it be a better practice to extend the `BroderType` types instead of hard coding the prop values?

Reply:
I was inclined to do this but saw other areas that used `border.color` so I can add colors to extend `BorderType` 
Would you want me to change all of the other `border` areas to  be `BorderType` ?

Reply:
I think what can be done instead is such:
```
border?: BorderType;
```
BorderType already includes the necessary typings for color, position, etc

Reply:
Your right i was making it to complicated 😓 I just added the `error.color` to the  `border?:`